landscape/client/manager/scriptexecution.py

```diff
@@ -105,6 +106,11 @@ def _run_script(self, filename, uid, gid, path, env, time_limit):
             uid = None
         if gid == os.getgid():
             gid = None
+        env = {
+            key: value.encode(sys.getfilesystemencoding(), errors='replace')
+            if isinstance(value, unicode) else value
```

Comment:
nitpick: "ifs" in list comprehension value reads weird. Parentheses and indentation would help.

Reply:
Fair enough - will push an update